observability/metric.go

```diff
@@ -0,0 +1,15 @@
+package observability
+
+import "go.uber.org/zap"
+
+var (
+	SecondsHistogramBuckets = []float64{0, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10}
```

Comment:
I'm not trying to nitpick, just curious: when measuring delay/latency, can any value ever be placed on the `0` bucket?

Reply:
It can be kept and we can revisit if we think it's impeding visualization.